s3-unpack/app/blob.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
+
+import io
+import logging
+
+from azure.storage.blob import BlobClient
+from azure.core.exceptions import ResourceNotFoundError
+
+DEFAULT_BUFFER_SIZE = 4 * 1024 ** 2
```

Comment:
```suggestion
DEFAULT_BUFFER_SIZE = 32 * 1024 ** 2
```
Tenker det er en grei default buffer, 32 MiB burde holde en god del filer inni tarfilen